Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -160,7 +109,7 @@ else
 	
         Write-Output "Copy will be Started for - $machine"
 
-        $job = Start-Job -ScriptBlock $CopyJob -ArgumentList $environmentName, $envOperationId, $machine, $sourcePath, $targetPath, $machineUserName, $machinePassword, $cleanTargetBeforeCopy
+        $job = Start-Job -ScriptBlock $CopyJob -ArgumentList $environmentName, $envOperationId, $machine, $sourcePath, $targetPath, $machineUserName, $machinePassword, $cleanTargetBeforeCopy, $connection
 
         $Jobs.Add($job.Id, $machine)
     
```

Comment:
aren't we getting the machineName in the output (deploymentResponse)? if that is the case should we open a bug at the DTL SDK layer? We are maintaining a hash table here with job ID's and machineNames which seems wrong to me.